XenAdmin/Wizards/DRWizards/DRFailoverWizardStoragePage.cs

```diff
@@ -332,15 +332,10 @@ private List<FibreChannelDevice> FiberChannelScan()
 
             Host master = Helpers.GetMaster(Connection);
 
-            if (master != null && (Helpers.HostBuildNumber(master) >= 9633
-                                || Helpers.HostBuildNumber(master) == Helpers.CUSTOM_BUILD_NUMBER))
+            if (master != null)

```

Comment:
I think we don't need to resolve the master here. If the master is null, we have greater problems than the SCSIId being added to the DeviceConfig.